kibana.md

```diff
@@ -1125,7 +1125,13 @@ readonlyrest_kbn.auth:
 
 ### Client Authentication Methods
 
-You can configure how the ReadonlyREST Kibana plugin sends `client_id` and `client_secret` to the identity provider using the option `readonlyrest_kbn.auth.<YOUR_OIDC_CONFIG>.tokenEndpointAuthMethod`. There are two available methods for authentication:
+You can configure how the ReadonlyREST Kibana plugin sends `client_id` and `client_secret` to the identity provider using the option 
+
+```
+readonlyrest_kbn.auth.oidc_kc.tokenEndpointAuthMethod: 'client_secret_post' // Available options: client_secret_basic (default) or client_secret_post 
```

Comment:
maybe it'd be better to use the "dot-less" syntax?
```yaml
readonlyrest_kbn:
  auth:
    oidc_kc:
      tokenEndpointAuthMethod: 'client_secret_post' // Available options: client_secret_basic (default) or client_secret_post
      [...]
```

to show that this is not a separate setting, but belong to some group of settings

Reply:
Sounds good, changed ✔️